app/store/index.js

```diff
@@ -1,9 +1,8 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-import initialState from 'app/initial_state';
 import configureAppStore from './store';
 
-const store = configureAppStore(initialState);
+const store = configureAppStore();
```

Comment:
why this ?

Reply:
`initialState` no longer needed by the store now. Only needed by the middleware when actioned to reset initial state.

Reply:
I’m really not sure about this, can you support the claim?

Reply:
Correction: `initialState` *is* needed by the store, but its use in store setup is delegated onward to `mattermost-redux`.

With store resetting logic now moved away from "store responsibility" to "middleware responsibility", the previous explicit use of `initialState` is now redundant. I'm 0/5 on who imports `initialState` to pass to `mattermost-redux`, so makes sense to let the parent pass down the initial state, just as before. Done via 0587e37992103a016985151b6af6a43d58bb6fda .

Reply:
thanks for adding this back.. yes `mattermost-redux` has an initial store, but this one has some view state the the one in mm-redux does not have and is important to initialize I think.

After the removal of mm-redux dependency we'll just have the one